package.json

```diff
@@ -26,7 +26,8 @@
     "db:setup": "./src/bin/initialize && yarn run db:migrate",
     "db:reset": "./src/bin/reset",
     "spec": "./node_modules/.bin/ava ava test --tap | tap-nyan",
-    "test": "nyc yarn spec"
+    "test": "nyc yarn spec",
+    "build:test": "yarn run db:setup && yarn run test"
```

Comment:
`run db:setup` initialising DB is supposed to be slower, but in this case it is running only once in travis and the extra over head is may be 1sec. 
There are arguments about using db:setup as a not "clean" solution but it would lead to race-conditions in creating DB. To solve that, we might have to write a script to wait and retry, IMO thats the worst!!. So I choose an unclean solution vs the worst!